test/jdk/javax/swing/JFileChooser/6698013/bug6698013.java

```diff
@@ -45,13 +46,26 @@ public class bug6698013 extends JApplet {
     final static VirtualFile subdirFile = new VirtualFile("testdir/subdir/subtest.txt", false);
 
     public static void main(String[] args) throws Exception {
-        SwingUtilities.invokeAndWait(() -> new bug6698013().init());
-    }
-
-    public void init() {
-        JFileChooser chooser = new JFileChooser(new VirtualFileSystemView());
-        chooser.setCurrentDirectory(root);
-        chooser.showOpenDialog(null);
+        String instructions = """
+                1. Go into 'subdir' folder via double click
+                2. Return to parent directory
+                3. Go into 'subdir' folder: select 'subdir' folder and press the 'Open' button""";
+
+        PassFailJFrame pfframe = PassFailJFrame.builder()
+                .title("bug6698013")
+                .instructions(instructions)
+                .rows(25)
```

Comment:
```suggestion
                .rows((int) INSTRUCTIONS.lines().count() + 2)
```

Your instructions frame has a lot of white space. Maybe worth incorporating this so you don't have to worry about adjusting the row count for each test.